spec/axe.spec.js

```diff
@@ -198,6 +198,51 @@ fractalLoad.then(() => {
           return cdp.Emulation.setDeviceMetricsOverride(SMALL_DESKTOP)
             .then(() => runAxe(cdp));
         });
+
+        if (process.env.ENABLE_SCREENSHOTS) {
+          const ROOT_DIR = `${__dirname}/..`;
+          const REL_DIR = `screenshots`;
+          const ABS_DIR = `${ROOT_DIR}/${REL_DIR}`;
+          const REL_PATH = `${REL_DIR}/${item.handle}.png`;
+          const ABS_PATH = `${ROOT_DIR}/${REL_PATH}`;
+          let doCompare = fs.existsSync(ABS_PATH);
+
+          it(doCompare ? 'is identical to previous screenshot'
+                       : 'saves screenshot for comparison later', () => {
+            const metrics = JSON.parse(JSON.stringify(SMALL_DESKTOP));
+            const { Page, Emulation } = cdp;
+
+            metrics.fitWindow = true;
+
+            return Emulation.setDeviceMetricsOverride(metrics)
+              .then(() => Page.getLayoutMetrics())
+              .then((result) => {
+                metrics.height = result.contentSize.height;
+                return Emulation.setDeviceMetricsOverride(metrics);
+              })
+              .then(() => Page.captureScreenshot({ format: 'png' }))
+              .then(result => {
+                const data = Buffer.from(result.data, 'base64');
+
+                if (doCompare) {
+                  const goldenData = fs.readFileSync(ABS_PATH);
```

Comment:
Oh, I've sometimes heard of this notion of "the expected file contents" as being called a "golden file". For example, it looks like Go's test suite for `gofmt` uses files with a `.golden` extension to contain the expected output of various calls and stuff:

https://golang.org/src/cmd/gofmt/gofmt_test.go

That said, it's confusing out of context, especially since I don't use the word "golden" anywhere else in this code, so I'll change it to something else like `previousData`.